### ISSUE TITLE
Issue with hardware serial of D1 mini v4.0

### ISSUE THREAD

**Issue by hackerspy21** [wemos/docs#36]
When connecting hardware serial TX to the RX of another D1 mini board, it stops sending data but with its loop still be active. Data are not received from the second board. Is there any significant change to previous versions in the hardware serial of the board?

**Comment by noega33** [comment]
Yes there are significant changes that prevent hardware serial from working.
Check my post: (at the end of the page)
https://github.com/probonopd/WirelessPrinting/issues/133
